### DISCUSSION TITLE
Banned

### DISCUSSION THREAD

**Discussion by Jeiel0rbit** [Sketchware-Pro/Sketchware-Pro#838] [General]
Please remove ban from more than two years ago: ```Jetronix⚡#7173``` I can't access Discord (IDE - Sketchware Pro)

**Comment by j-px** [comment]
There's no such user in the Banned list. Maybe send your account's ID (a really long number), and I can try to unban your account using that, or at least see what the ban reason was.

**Reply by Jeiel0rbit** [reply]
```1014151811583709245``` (ID)

**Reply by j-px** [reply]
Just successfully banned and unbanned your account. It shouldn't be banned anymore, but it also doesn't seem to have been before this.

**Reply by Jeiel0rbit** [reply]
Looks like I'm banned or the link doesn't work: https://discord.gg/Dc8ZDBRK5V

**Reply by j-px** [reply]
The invite link works for me and is known to Discord as a valid one. [Maybe try this one](https://discord.gg/kq39yhT4rX).

**Reply by Jeiel0rbit** [reply]
Ok, I'm still banned. Unfortunately even Discord cannot help me in this case.

**Reply by nicesapien-too** [reply]
Create a new discord account or try to join with a vpn and same account

**Reply by Jeiel0rbit** [reply]
> Create a new discord account or try to join with a vpn and same account

This is a new account!
